src/main/resources/META-INF/mods.toml

```diff
@@ -32,3 +32,9 @@ See where you go
    versionRange="[5.9.8,)"
    ordering="NONE"
    side="BOTH"
+[[dependencies.hudcompass]]
+   modId="curios"
+   mandatory=false
+   versionRange="[${curios_version},)"
```

Comment:
This isn't going to work. I don't use gradle replacement variables in my mods.toml (no process resources). It should also be set to whichever version is the earliest compatible, not blindly set it to the version it was built with.